### ISSUE TITLE
Assigning Custom PDB to PostgreSQL cluster

### ISSUE THREAD

**Issue by ManjunathCV91** [zalando/postgres-operator#2196]
I would like to know if there is any option provided as part of Postgres-Operator to configure custom PDB and assign it to the cluster?

Documentation states only option available is to enable or disable a PDB.

**image:** registry.opensource.zalan.do/acid/postgres-operator:v1.8.2

**Comment by FxKu** [comment]
Yes, you cannot customize the PDB. You could only change the generated one temporarily until the operator changes it back on the next sync sycle.
